app/views/store/homepage.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1>Welcome
+<% if user_signed_in? %>
```

Comment:
let's fix indentation and add the if inside h1 so there is only one closing h1 tag

Reply:
fixed